CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+## Table of Contents
+
+- [Changelog](#changelog)
+  - [[5.0.0] - 2024-04-23](#500---2024-04-23)
+    - [Changed](#changed)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Changelog
+
+All notable changes to the Transcend CLI tools will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [5.0.0] - 2024-04-23
+
+### Changed
+
+- Added logic to decrypt encrypted request identifiers to `cli-manual-enrichment-pull-identifiers`,
```

Comment:
could we expand this to show some before and after examples for each of the 3 commands? we also need to explain the nuance for each command, e.g. the sombraAuth argument is only needed when self hosting sombra, and for the cli-request-restart function, its only needed if self hosted and using the `--copyIdentifiers` argument

the names here also don't align with the customer facing commands. the customer knows these by `tr-request-export` etc not `cli-request-export`